requirements.txt

```diff
@@ -1,5 +1,5 @@
-boto3==1.20.26
-botocore==1.23.54
+boto3>=1.34.0
+botocore>=1.34.0
```

Comment:
Looks like latest version is 1.40.x - might be worth updating to stay up to date?

https://pypi.org/project/boto3/
https://pypi.org/project/botocore/